src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/custom.py

```diff
@@ -699,6 +749,67 @@ def elastic_pool_update(
     return instance
 
 
+class ElasticPoolCapabilitiesAdditionalDetails(Enum):
+    max_size = 'max-size'
+    db_dtu_min = 'db-dtu-min'
+    db_dtu_max = 'db-dtu-max'
+    db_max_size = 'db-max-size'
+
+
+def elastic_pool_list_capabilities(
+        client,
+        location,
+        edition=None,
+        dtu=None,
+        detail=None):
+
+    # Fixup parameters
+    if detail is None:
+        detail = []
+    if dtu is not None:
+        dtu = int(dtu)
+
+    # Get capabilities tree from server
+    capabilities = client.list_by_location(location)
+
+    # Get subtree related to elastic pools
+    editions = next(sv for sv in capabilities.supported_server_versions
+                    if sv.name == _default_server_version).supported_elastic_pool_editions
+
+    # Filter by edition
+    if edition is not None:
+        editions = [e for e in editions if e.name.lower() == edition.lower()]
+
+    # Filter by dtu
+    if dtu is not None:
+        for e in editions:
+            e.supported_elastic_pool_dtus = [
+                d for d in e.supported_elastic_pool_dtus
+                if d.limit == dtu]
+
+    # Remove editions with no service objectives (due to filters)
+    editions = [e for e in editions if len(e.supported_elastic_pool_dtus) > 0]
```

Comment:
simply do `if e.supported_elastic_pool_dtus` is enough. The truthiness of an empty collection is false.